docs/UserGuide.md

```diff
@@ -348,6 +381,7 @@ Settles the amount owed by the student and adds it to the paid amount. To record
 * The **INDEX** refers to the index number shown in the displayed student list.
 * The **INDEX** **must be a positive integer** 1, 2, 3, …​
 * **AMOUNT** must be a positive value and must not be more than **OWED_AMOUNT**.
```

Comment:
For this one right `* **AMOUNT** must be a positive value and must not be more than **OWED_AMOUNT**.`, can you rephrase it to

`* **AMOUNT** must be a positive value and must not be more than the current amount owed by the student`